### PR TITLE
HttpJson collector: Allow usage of header configuration

### PR DESCRIPTION
**Pull request by eMerzh**
I wanted to be able to specify some headers on my httpjson collector,
as in the http one ...here is my small patch about it ..

It's useful for authorisation, for Host check, and plenty other things

### CODE REVIEW

File: src/collectors/httpjson/httpjson.py [hunk 1]
```diff
@@ -27,7 +28,8 @@ def get_default_config(self):
         default_config = super(HTTPJSONCollector, self).get_default_config()
         default_config.update({
             'path': 'httpjson',
-            'url': 'http://localhost/stat'
+            'url': 'http://localhost/stat',
+            'headers': {'User-Agent': 'Diamond HTTP collector'},
```
Comment (kormoc) [inline src/collectors/httpjson/httpjson.py]: Does this work as expected for setting in config files?

**Comment by eMerzh** [comment]
now, it should be better ... i left a small typo ... 
but it seems to work as expected ...no?

**Comment by eMerzh** [comment]
Any news on that?

**Comment by bmhatfield** [comment]
Hi - This PR appears to have PEP8 issues. Would you mind cleaning that up?

**Comment by eMerzh** [comment]
@bmhatfield you're right :) here it is ... :+1:

File: src/collectors/httpjson/httpjson.py (hunk 1 above)
Comment (bmhatfield) [inline src/collectors/httpjson/httpjson.py]: I'm pretty happy with this change in general / prepping to merge this, but I'd like to second this question. Can you provide an example configuration snippet to confirm what it looks like?

Comment (eMerzh) [inline src/collectors/httpjson/httpjson.py]: If you set 

```

[[HTTPJSONCollector]]
enabled = True
measure_collector_time = False
url = http://localhost/HelloWorld
[[[headers]]]
X-Hello=World 
```
it should work as expected (set a header key: X-Hello and value World)

**Comment by bmhatfield** [comment]
Thanks for this!

**Pull request merged**